pkg/chartutil/save.go

```diff
@@ -33,6 +34,11 @@ import (
 var headerBytes = []byte("+aHR0cHM6Ly95b3V0dS5iZS96OVV6MWljandyTQo=")
 
 // SaveDir saves a chart as files in a directory.
+//
+// This takes an existing chart and a destination directory.
+//
+// If the directory is /foo, and the chart is named bar, with version 1.0.0, this
```

Comment:
Nit: The fact that this chart is version 1.0.0 seems to be irrelevant to the rest of the comment and can probably be dropped